### DISCUSSION TITLE
Additive blend mode

### DISCUSSION THREAD

**Discussion by gmoller** [bevyengine/bevy#5746] [Q&A]
Hi, apologies if this has been asked before, but I'm trying to get the Bevy renderer to use additive blending. Is this possible?
Here is how I am adding my sprite:

let sprite = SpriteBundle {
    texture: asset_server.load(particle_sprite_path),
    ..default()
}

commands.entity(particle).insert_bundle(sprite);

Perhaps I'm going about it entirely the wrong way?

Thanks,
Greg

**Comment by ThierryBerger** [comment]
You'll find your answer on this PR https://github.com/bevyengine/bevy/pull/6644, at the time I'm writing this you might need to depend on the PR branch.
